test/lit/passes/gsi-debug.wast

```diff
@@ -37,21 +37,9 @@
   ;; CHECK-NEXT:  ;;@ drop.c:10:1
   ;; CHECK-NEXT:  (drop
   ;; CHECK-NEXT:   ;;@ struct.c:20:2
-  ;; CHECK-NEXT:   (select
-  ;; CHECK-NEXT:    ;;@ struct.c:20:2
-  ;; CHECK-NEXT:    (i32.const 42) (; i32 ;)
-  ;; CHECK-NEXT:    ;;@ struct.c:20:2
-  ;; CHECK-NEXT:    (i32.const 1337) (; i32 ;)
-  ;; CHECK-NEXT:    ;;@
-  ;; CHECK-NEXT:    (ref.eq
-  ;; CHECK-NEXT:     ;;@
-  ;; CHECK-NEXT:     (ref.as_non_null
-  ;; CHECK-NEXT:      ;;@ local.c:30:3
-  ;; CHECK-NEXT:      (local.get $struct) (; (ref null $struct) ;)
-  ;; CHECK-NEXT:     ) (; (ref $struct) ;)
-  ;; CHECK-NEXT:     ;;@
-  ;; CHECK-NEXT:     (global.get $global1) (; (ref $struct) ;)
-  ;; CHECK-NEXT:    ) (; i32 ;)
+  ;; CHECK-NEXT:   (struct.get $struct 0
+  ;; CHECK-NEXT:    ;;@ local.c:30:3
+  ;; CHECK-NEXT:    (local.get $struct) (; (ref null $struct) ;)
```

Comment:
This output no longer contains the `select` it intends to contain. Same with most of the other GSI tests. I will investigate.